### PR TITLE
feat: scarb version

### PR DESCRIPTION
**Pull request by suiwater**
add scarb version
add Sierra language

- - -
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/665)

### CODE REVIEW

**Comment by TAdev0** [comment]
hey @suiwater. I'm not sure this modification is useful, you are just writing twice the same thing. It seems better to explain how to install with ASDF,  then with Scarb, an finally show the  output version which will be the same with both methods.

But scarb 2.6.0 was just released yesterday, therefore you can modify all mentions of scarb version 2.5.3 in the book with 2.6.0 if you want :)

File: src/ch01-01-installation.md [hunk 1]
```diff
@@ -42,18 +42,26 @@ and set a global version:
...
+```
+
 Otherwise, you can simply run the following command in your terminal, and follow the onscreen instructions. This will install the latest stable release of Scarb.
 
 ```bash
 curl --proto '=https' --tlsv1.2 -sSf https://docs.swmansion.com/scarb/install.sh | sh
 ```
-
-- In both cases, you can verify installation by running the following command in a new terminal session, it should print both Scarb and Cairo language versions, e.g:
+- You can verify installation by running the following command in a new terminal session, it should print both Scarb, Cairo and Sierra language versions, e.g:
```
Comment (enitrat) [inline src/ch01-01-installation.md:59]: this is a repetition from the text above

Comment (suiwater) [reply src/ch01-01-installation.md:59]: yes

**Comment by suiwater** [comment]
https://github.com/cairo-book/cairo-book/issues/666
close it.

**Pull request closed without merging**